.github/actions/collect-test-data/action.yml

```diff
@@ -25,11 +25,11 @@ runs:
   using: 'composite'
   steps:
     - name: Collect test data
-      uses: getsentry/action-collect-test-data@c0a2fd3f79d3867e2c8b486ccb1c4bf925477742 # v0.3.0
+      uses: getsentry/action-collect-test-data@5459fd73037103ba4f1a39eb24e6f8722b92de14 # v0.4.0
       with:
         path: ${{ inputs.artifact_path }}
         gcs_path: ${{ inputs.gcs_bucket }}/pytest/${{ github.repository }}/${{ github.workflow }}/${{ github.job }}${{ inputs.matrix_instance_number != '' && '/instance' || '' }}${{ inputs.matrix_instance_number != '' && inputs.matrix_instance_number || '' }}/${{ github.repository_id }}/${{ github.run_id }}/${{ github.run_attempt }}
         gcp_project_id: ${{ inputs.gcp_project_id }}
         workload_identity_provider: ${{ inputs.workload_identity_provider }}
         service_account_email: ${{ inputs.service_account_email }}
-      continue-on-error: true
+      continue-on-error: false # This data is important.
```

Comment:
we should not break CI if collection fails

Reply:
I'll file a backlog for alerting.
I so very much don't trust alerting though :(